### ISSUE TITLE
Lots of blank space in meta data?

### ISSUE THREAD

**Issue by daviddalzell** [fnordware/SuperPNG#9]
I noticed that there is a lot of blank space in the meta data tEXt chunk  after </x:xmpmeta>  tag and before .<?xpacket end="w"?>  I counted 12048 bytes divided into 64-byte long strings each ending with a .

**Comment by fnordware** [comment]
Yeah, there could be. It's whatever Adobe feels like putting in there.  You can take a look by going File > File Info > Raw Data.

**Comment by daviddalzell** [comment]
I can only see the blanks in a hex viewer -- it's annoying and adobe-like // related topic: have you tried using a zTXt block in Photoshop or AE files?

**Comment by fnordware** [comment]
I have not really messed with zTXt compression. The [XMP spec](http://wwwimages.adobe.com/content/dam/Adobe/en/devnet/xmp/pdfs/XMP%20SDK%20Release%20cc-2016-08/XMPSpecificationPart3.pdf) says it should be uncompressed. If you can build the plug-in, you can mess around with which chunks you're saving.
